lerobot-web-be/app/utils/robots.py

```
@@ -0,0 +1,106 @@
+):
+    prefixes = get_serial_prefixes()
+    if robot_type == RobotType.SO100:
+        if is_bi_setup:
+            follower_config = BiSO100FollowerConfig(
+                left_arm_port=f"{prefixes[0]}{follower_map['left']}",
+                right_arm_port=f"{prefixes[0]}{follower_map['right']}",
+                cameras=camera_config,
+            )
+            return BiSO100Follower(follower_config)
+        else:
+            follower_id = list(follower_map.values())[0]
```

Ruff (0.14.3)

Prefer `next(iter(follower_map.values()))` over single element slice

Replace with `next(iter(follower_map.values()))`

(RUF015)